.github/workflows/run-tests.yml

```diff
@@ -234,9 +241,18 @@ jobs:
           echo "Building Wire-iOS..."
           xcodebuild build -workspace wire-ios-mono.xcworkspace -scheme Wire-iOS -destination 'platform=iOS Simulator,OS=${{ env.IOS_VERSION }},name=${{ env.IPHONE_MODEL }}' | xcpretty
           echo "Testing Wire-iOS..."
-          xcodebuild test -retry-tests-on-failure  -test-repetition-relaunch-enabled YES -workspace wire-ios-mono.xcworkspace -scheme Wire-iOS -destination 'platform=iOS Simulator,OS=${{ env.IOS_VERSION }},name=${{ env.IPHONE_MODEL }}' | xcpretty --report junit --screenshots --output build/reports/Wire-iOS.junit
+          xcodebuild test -workspace wire-ios-mono.xcworkspace -scheme Wire-iOS -destination 'platform=iOS Simulator,OS=${{ env.IOS_VERSION }},name=${{ env.IPHONE_MODEL }}' | tee xcodebuild.log | xcpretty --report junit --output build/reports/Wire-iOS.junit
```

Comment:
1. note -retry-tests-on-failure doesn't quite work for snapshot tests as it will append the number of try to the expected snapshot reference. Since it will not be there it will fail.

One way to avoid this would be to override the filename or move the tests to separate target to run with different args

2. xcpretty does not show the crash that can happen during runs hence the new xcodebuild.log